test/tstDetailsKokkosExtKernelStdAlgorithms.cpp

```diff
@@ -41,13 +45,11 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(nth_element, DeviceType, ARBORX_DEVICE_TYPES)
   {
     int const n = v_ref.size();
 
-    Kokkos::View<float *, DeviceType> v("v", n);
-    Kokkos::deep_copy(
-        space, v,
-        Kokkos::View<float *, Kokkos::HostSpace,
-                     Kokkos::MemoryTraits<Kokkos::Unmanaged>>(v_ref.data(), n));
+    Kokkos::View<float *, DeviceType> v("Testing::v", 0);
```

Comment:
I thought we were going for "[ArborX]{Test,Example}::foo"

Reply:
We use `"Testing::foo` and `"Example::foo`. We use `ArborXTest` namespaces.

Reply:
Please Test~~ing~~ (does not need to be resolved here

Reply:
> Please Test~ing~ (does not need to be resolved here

Are you asking to change all `Testing::` to `Test::` in a separate PR? Or to `ArborXTest::`?